src/Engine/ProtoCore/FFI/ImportModuleHandler.cs

```diff
@@ -106,9 +106,7 @@ public ProtoCore.AST.AssociativeAST.ImportNode Import(string moduleName, string
                     }
                     _coreObj.BuildStatus.LogSemanticError(sb.ToString());
                 }
-                if (ex.InnerException != null)
-                    _coreObj.BuildStatus.LogSemanticError(ex.InnerException.Message);
-                _coreObj.BuildStatus.LogSemanticError(ex.Message);
+                _coreObj.BuildStatus.LogSemanticError(ex.ToString());
```

Comment:
What about the inner exception message?

Reply:
@aparajit-pratap `exception.ToString()` will log the innerException as well. It's documented as:

```
The default implementation of [ToString](https://learn.microsoft.com/en-us/dotnet/api/system.exception.tostring?view=net-7.0) obtains the name of the class that threw the current exception, the message, the result of calling [ToString](https://learn.microsoft.com/en-us/dotnet/api/system.exception.tostring?view=net-7.0) on the inner exception, and the result of calling [Environment.StackTrace](https://learn.microsoft.com/en-us/dotnet/api/system.environment.stacktrace?view=net-7.0). If any of these members is null, its value is not included in the returned string.
```